packages/microcosm/src/domain.js

```diff
@@ -85,10 +88,16 @@ export class Domain<State: mixed = null> extends Agent {
   }
 
   _resolve(action: Subject): DomainHandler<State>[] {
-    switch (action.toString()) {
-      case String(RESET):
-      case String(PATCH):
```

Comment:
Doing some benchmarks on my scheduler branch, wrapping a String like this was a major contributor to JS execution time.